docs/vc-http-api.yml

```diff
@@ -41,6 +41,28 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /credentials/updateStatus:
+    post:
+      tags:
+        - Issuer
+      summary: Updates the status of an issued credential
+      operationId: updateCredentialStatus
```

Comment:
https://restfulapi.net/resource-naming/

note that the operation has the word "update" but the route should not... this route is an example of a `controller` style endpoint.